examples/deploy-container.sh

```diff
@@ -8,10 +8,12 @@ set -euo pipefail
 # Run with option `--interactive` or `-i` to start a shell for interacting with
 # the node.
 
-if [[ ! -v IN_NIX_SHELL ]]; then
+if [[ ! -v NIX_BITCOIN_EXAMPLES_DIR ]]; then
     echo "Running script in nix shell env..."
     cd "${BASH_SOURCE[0]%/*}"
     exec nix-shell --run "./${BASH_SOURCE[0]##*/} $*"
+else
+    cd "$NIX_BITCOIN_EXAMPLES_DIR"
```

Comment:
Maybe include a comment explaining why, for people who are writing their "own experiments" from this example.

Reply:
I think this is unnecessary because there's only one possible reason for changing the working directory: Because we're using paths relative to this dir. And that's what happens below.
How would you phrase the comment?

Reply:
It's not strictly necessary, but maybe something like

```
# Move into `examples` directory from arbitrary working dir
```

Reply:
In this case, the comment repeats WHAT the code does. 
But I really think the WHAT and also the WHY (see above) of `cd ...`  are self-explanatory and it's okay to leave it as is.